vcpkg.json

```diff
@@ -19,8 +11,7 @@
     },
     {
       "name": "gdal",
-      "default-features": false,
-      "features": ["recommended-features"]
```

Comment:
this also pulls in a lot of unneeded dependencies. `libgeotiff` is a hard dependency, should work for our use case